views/applications/all.html

```diff
@@ -0,0 +1,49 @@
+{% extends "_donut.html" %}
+
+{% block pageHead %}
+  {{ super() }}
+{% endblock %}
+
+{% block header %}
+  {{ super() }}
+{% endblock %}
+
+{% block main %}
+  <div class="introduction">
+    <h1 class="introduction--h1">{{structure.name}}</h1>
+    <p class="introduction--p">{{structure.description}}</p>
+    <a href="/${config.base}/${config.actions.add}" class="introduction--button">{{config.actions.add|capitalize}} {{config.base}}</a>
+  </div>
+
+  {% if applications %}
+  <table class="base--table">
+    <thead class="base--thead">
+      <tr class="base--tr">
+        <th class="base--th">Application</th>
+        <th class="base--th">Created</th>
+        <th class="base--th">Live</th>
+        <th class="base--th">Updated</th>
+        <th class="base--th">Sunset</th>
+        <th class="base--th">Edit</th>
+      </tr>
+    </thead>
+    <tbody class="base--tbody">
+    {% for a in applications %}
+      <tr class="base--tr">
+        <td data-th="Application" class="base--td">{{a.name}}</td>
+        <td data-th="Created" class="base--td">{{a.created}}</td>
+        <td data-th="Live" class="base--td">{{a['live-endpoint']}}</td>
```

Comment:
## urls
from #115:
```
      And I can see if they have a go live endpoint registered
      And I can see if they have an update endpoint registered
      And I can see if they have a sunset endpoint registered
```
nothing about pass/fail of recent attempt. These would be blank if empty, and the URL if not empty. Would you like it to rewrite the scenario to clarify? Or leave this as is and write a new scenario to add these features once the base version of this is complete?

## variables names
follows the variables set out [in the merged schema](https://github.com/punchcard-cms/punchcard/blob/master/lib/database/schemas.js#L137). I can change these, but would you like a schema change too?

## link
Only instruction in #115 is `And I will have a link to edit the application`. Is there a different story I'm missing for this?

## run views
What's a run view? Scenario for viewing and editing a single application:

```gherkin
  Scenario: View One
    Given that I have an application registered
     When I navigate to /applications/:id
     Then I can see the application's name
      And I can see each of the application's endpoints
      And both the Client ID and Client Secret will be displayed
      And I can reset the Client Secret
      And I can update the application's name or endpoints
      And I can delete the application
```